scripts/sync.sh

```diff
@@ -242,7 +242,8 @@ function format {
 	local -r repo_path=$1
 	(
 		cd "$repo_path"
-		npx prettier --write "*.md" .github
+		npm install ory-prettier-styles@1.3.0
```

Comment:
Please remove this, we no longer use this preset!

Reply:
We don't? What do we do instead? Do we have some kind of announcement for this? Iirc we just reformatted all of our code with this preset.

Reply:
No, we used to use this preset. Then we deprecated it in favor of the default prettier styles. That was part of the reformatting, removing the preset, not adding it.